client/src/views/files/FoldersPage.vue

```diff
@@ -1324,25 +1334,44 @@ async function openGlobalContextMenu(event: Event): Promise<void> {
 
 async function openEntryContextMenu(event: Event, entry: EntryModel, onFinished?: () => void): Promise<void> {
   const selectedEntries = getSelectedEntries();
+  let data: any;
+
+  if (isLargeDisplay.value) {
```

Comment:
This function could be moved to another file, the FoldersPage is already gigantic. A function that returns `FileAction | undefined`